### ISSUE TITLE
Translation for the Secret Recovery Phrase update

### ISSUE THREAD

**Issue by omnat** [MetaMask/metamask-mobile#2665]
## Translate the updated text

With 2.3 update: We update Seedphrase (and inconsistent variations like seed phrase, Seed Phrase, Seed phrase, seedphrase) to Secret Recovery Phrase in English. AND the new video was added to 2.3.

Translation for this update (to release in 2.4): In the all the areas of the app where this change was made, we should update the translation file(s) for all the 10 languages we support.

## Translate the video

For 2.4: Add subtitles file to the video in all the languages we support (including English, and the 10 other languages we support)

## Announcement

* Marketing to send an announcement for the change of terminology from 'Seedphrase' to 'Secret Recovery Phrase', the motivation behind it. 

* In-app "What's new" update for this announcement
** We need copy for this, and need to send this copy for translation (via Jacob)

(NOT related to #2651)

**Comment by omnat** [comment]
@jacobcantele could we ask the translation company for the following files to be translated in the 10 languages we support? And get the estimated ready time from them?

1. Mobile diff file ( @rickycodes to send it to you)
2. Subtitles file (Same as extension subtitles file)

**Comment by mobularay** [comment]
@estebanmino brings up other risk: if someone move video location in Github, then this will break. How do we deal with this UX?! 

Ah, we still have need to talk about the how-to-implement!

**Comment by mobularay** [comment]
Lionbridge has received the differential file and video to translate into the 9 non-English languages & video subtitles. ETA of tunaround time is between May 18 Tues and 20 Thurs.

**Comment by mobularay** [comment]
List of tangible items required before this ticket is workable:

- [x] [translated seed phrase update diff](https://consensys.slack.com/archives/C021Q4NB33M/p1621305463001400)
- [x] translated subtitle files for seed phrase explainer video (same zip file as above)
- [x] [English copy of 'What's new' in-app announcement](https://consensys.slack.com/archives/C021Q4NB33M/p1621035458037300)
- [ ] translated 'What's new' in-app announcement copy
- [ ] set in-app announcement to remind users about SRP every 3 months??? 

@omnat can you double-check the above list and see if it is accurate? 
At this moment, I am somehow unable to download the zip file from the Slack channel, so I am linking the Slack thread that contains the translated files

**Comment by mobularay** [comment]
As discussed in May 20 Sprint Grooming session @omnat @rickycodes 

In-Scope Technical Requirements:
- [x] Host video & subtitles in Github for now
- [x] Use same media player component that @cjeria has designed for NFT collectibles

Out-of-Scope:
- [ ] Long term better video & subtitles hosting location
- [ ] Better translation process as current process is error-prone (e.g. getting the diff code, translators review Github PRs, sending files via Slack channel, etc)

**Comment by mobularay** [comment]
This work is likely delayed to v2.6 (June 14) due to [LionBridge's delay](https://consensys.slack.com/archives/C021Q4NB33M/p1622073442028100?thread_ts=1621348628.013100&cid=C021Q4NB33M) CC @omnat

**Comment by sethkfman** [comment]
Currently blocked by a bug in `react-native-video` with our current version of react-native not being supported. The subtitles are not able to be loaded or shown during the video. 

More work would be needed to patch or dive in to the native code to fix the issue. I suggest splitting this portion of the story out for future work. 

CC: @omnat & @mobularay

**Comment by mobularay** [comment]
From Sprint Planning June 20:
- Design QA pass only for subtitles
- Need Design QA process the rest of the translated part (SRP and Swaps - how copy looks like on the UI)
    - Process: attach screenshots and labels to the PR if it's not merged yet

@jakehaugen @sethkfman ^^^
